src/update_variety_data.py

```diff
@@ -35,16 +35,27 @@
 
 def _create_variety_data(config):
     full_variety_path = os.path.join(get_src_dir(), config['data_storage']['variety_path'])
-    output, return_code = execute_shell_command_get_return_code(
-        'mongo --port {mongo_port} {main_database} -u "{username}" -p "{password}" --authenticationDatabase "admin" '
-        '--eval "var collection = \'file_objects\', persistResults=true" {script_path}'.format(
+    mongo_call = (
+        'mongo --port {mongo_port} -u "{username}" -p "{password}" --authenticationDatabase "admin" '.format(
             mongo_port=config['data_storage']['mongo_port'],
             username=config['data_storage']['db_admin_user'],
             password=config['data_storage']['db_admin_pw'],
-            main_database=config['data_storage']['main_database'],
+        )
+    )
+    output, return_code = execute_shell_command_get_return_code(
```

Comment:
should we abort if return_code is not `0`

Reply:
I don't know... that's pretty much all that happens in the script anyway

Reply:
And the return code is used as the exit code of the whole script. Of course we could add some error messages in case of an error. I tried to add as few lines as possible though because the script isn't tested.